support-frontend/assets/helpers/abTests/abtestDefinitions.js

```diff
@@ -35,53 +33,6 @@ export const tests: Tests = {
     seed: 5,
   },
 
-  fancyAddressTest: {
```

Comment:
is this just left over cleanup from the other test?

Reply:
Yup